### ISSUE TITLE
Release recent changes

### ISSUE THREAD

**Issue by nicolas2mey** [rhysd/actionlint#398]
Hi and first: thank you for this useful project 👍 

I see the latest release of the project is a little bit old

I meet issue regarding available input for `docker/build-push-action` returning me with the version `v1.6.26` that 
```
 input "secret-envs" is not defined in action "docker/build-push-action@v5"
```

with `main` version, the problem is not raised 

I think some changes fixed that but anyway

So my question is: 
> **Is it planned to make a new tag/release on the project ?**

**Comment by rhysd** [comment]
[1.6.27](https://github.com/rhysd/actionlint/releases/tag/v1.6.27) was shipped. I apologize for the delay.
